src/Pages/ConfessionPage.re

```diff
@@ -0,0 +1,147 @@
+//open Belt;
+
+type state = {
```

Comment:
this file is SUPER crowded. Having your reducer, actions and JSX all in one file is going to get hairy REAL fast. Typically you would split this stuff up in to separate files.